starboard/nplb/BUILD.gn

```diff
@@ -22,7 +23,19 @@ config("cast_codec_tests") {
   cflags = [ "ENABLE_CAST_CODEC_TESTS" ]
 }
 
-target(gtest_target_type, "nplb") {
+config("wnos") {
+  cflags = [
+    "-Wno-ignored-qualifiers",
+    "-Wno-self-assign-overloaded",
+    "-Wno-pessimizing-move",
+    "-Wno-unused-local-typedef",
+    "-Wno-unreachable-code",
+    "-Wno-unreachable-code-break",
+    "-Wno-enum-constexpr-conversion",
```

Comment:
Hmm, this flag is already present in the current files: https://source.corp.google.com/h/github/youtube/cobalt/+/main:starboard/nplb/BUILD.gn;l=269?q=%22-Wno-enum-constexpr-conversion%22&ss=h%2Fgithub%2Fyoutube%2Fcobalt%2F%2B%2Frefs%2Fheads%2Fmain

maybe we need to reuse that config ?

Reply:
same with -Wno-ignored-qualifiers
https://source.corp.google.com/h/github/youtube/cobalt/+/main:starboard/build/config/BUILD.gn;l=39?q=%22-Wno-ignored-qualifiers%22&ss=h%2Fgithub%2Fyoutube%2Fcobalt%2F%2B%2Frefs%2Fheads%2Fmain

Reply:
Maybe the existing configs need to be removed - upto you.